### PR TITLE
Ore balancing (revert from wizden nerf)

### PR DESCRIPTION
**Pull request by Lyacs**
## About the PR

I reverted back almost at the identical ore composition and ore procedural generation

## Why / Balance

Bounty : https://discord.com/channels/1202734573247795300/1313673129599438899

With the recent changes, it seems there is an heavy strain on station resources and science resources in general, mostly due to wizden #https://github.com/space-wizards/space-station-14/pull/30920.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- remove: Removed Wizden ore nerf
- tweak: Ore procedural generation on asteroïds and vg roid. Increased ore composition

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Materials/ore.yml [hunk 1]
```diff
@@ -370,7 +370,7 @@
   - type: Material
   - type: PhysicalComposition
     materialComposition:
-      RawSalt: 100
+      RawSalt: 300 # Goobstation - was 100
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Materials/ore.yml:373]: just remove salt chem couldnt give 2 shits about it, coal is worth infinitely more to chem than salt 
https://github.com/DeltaV-Station/Delta-v/pull/2027

**Comment by deltanedas** [comment]
still wont fix the problem of salv being shitters and not mining, they need bomb/shock implants that QM can remotely activate if they wont do anything

**Comment by Lyacs** [comment]
Totally agree  with you, I did talk about it with Achylle, but they just wanted this for the bounty. I see that a temporary fix for the lack of resources.

**Comment by Tropica1Owl** [comment]
> still wont fix the problem of salv being shitters and not mining, they need bomb/shock implants that QM can remotely activate if they wont do anything

I have nothing to do with this PR, but I had discussed with Mocho about how rewarding salvagers for mining would fix this issue, at least at a base level if the rewards are interesting. Yes, the proposed solution is, once more, "Copy SS13". Who would've figured out they solved those problems 20 years prior?

**Pull request merged**